[2022-08-06 19:37:21,598] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-05T00:00:00+00:00 [queued]>
[2022-08-06 19:37:21,619] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-05T00:00:00+00:00 [queued]>
[2022-08-06 19:37:21,620] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:37:21,620] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:37:21,620] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:37:21,656] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-05 00:00:00+00:00
[2022-08-06 19:37:21,669] {standard_task_runner.py:52} INFO - Started process 1327 to run task
[2022-08-06 19:37:21,673] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-05T00:00:00+00:00', '--job-id', '393', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmprv5mihf0', '--error-file', '/tmp/tmpzubbs0lf']
[2022-08-06 19:37:21,673] {standard_task_runner.py:80} INFO - Job 393: Subtask second_task
[2022-08-06 19:37:21,763] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-05T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:37:21,876] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-05T00:00:00+00:00
[2022-08-06 19:37:21,877] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:37:21,878] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:37:21,892] {subprocess.py:85} INFO - Output:
[2022-08-06 19:37:21,894] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:37:21,894] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:37:21,933] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211205T000000, start_date=20220806T193721, end_date=20220806T193721
[2022-08-06 19:37:21,966] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:37:22,011] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:51:14,235] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-05T00:00:00+00:00 [queued]>
[2022-08-06 21:51:14,253] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-05T00:00:00+00:00 [queued]>
[2022-08-06 21:51:14,254] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:51:14,254] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:51:14,254] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:51:14,272] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-05 00:00:00+00:00
[2022-08-06 21:51:14,280] {standard_task_runner.py:52} INFO - Started process 1511 to run task
[2022-08-06 21:51:14,284] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-05T00:00:00+00:00', '--job-id', '391', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpqi22h6w6', '--error-file', '/tmp/tmpov7kgb9a']
[2022-08-06 21:51:14,284] {standard_task_runner.py:80} INFO - Job 391: Subtask second_task
[2022-08-06 21:51:14,349] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-05T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:51:14,445] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-05T00:00:00+00:00
[2022-08-06 21:51:14,446] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:51:14,447] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:51:14,462] {subprocess.py:85} INFO - Output:
[2022-08-06 21:51:14,464] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:51:14,465] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:51:14,503] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211205T000000, start_date=20220806T215114, end_date=20220806T215114
[2022-08-06 21:51:14,535] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:51:14,595] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:56:37,103] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-05T00:00:00+00:00 [queued]>
[2022-08-06 21:56:37,113] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-05T00:00:00+00:00 [queued]>
[2022-08-06 21:56:37,113] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:56:37,114] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:56:37,114] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:56:37,128] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-05 00:00:00+00:00
[2022-08-06 21:56:37,134] {standard_task_runner.py:52} INFO - Started process 1296 to run task
[2022-08-06 21:56:37,136] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-05T00:00:00+00:00', '--job-id', '390', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp2l_jc44a', '--error-file', '/tmp/tmpiz030vm7']
[2022-08-06 21:56:37,136] {standard_task_runner.py:80} INFO - Job 390: Subtask second_task
[2022-08-06 21:56:37,204] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-05T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:56:37,280] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-05T00:00:00+00:00
[2022-08-06 21:56:37,281] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:56:37,281] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:56:37,292] {subprocess.py:85} INFO - Output:
[2022-08-06 21:56:37,293] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:56:37,293] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:56:37,318] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211205T000000, start_date=20220806T215637, end_date=20220806T215637
[2022-08-06 21:56:37,348] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:56:37,373] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:11:43,184] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-05T00:00:00+00:00 [queued]>
[2022-08-06 22:11:43,199] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-05T00:00:00+00:00 [queued]>
[2022-08-06 22:11:43,199] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:11:43,199] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:11:43,199] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:11:43,216] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-05 00:00:00+00:00
[2022-08-06 22:11:43,226] {standard_task_runner.py:52} INFO - Started process 1317 to run task
[2022-08-06 22:11:43,230] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-05T00:00:00+00:00', '--job-id', '393', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp3yf2yvm9', '--error-file', '/tmp/tmpqwl8sk01']
[2022-08-06 22:11:43,231] {standard_task_runner.py:80} INFO - Job 393: Subtask second_task
[2022-08-06 22:11:43,312] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-05T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:11:43,399] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-05T00:00:00+00:00
[2022-08-06 22:11:43,400] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:11:43,401] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:11:43,414] {subprocess.py:85} INFO - Output:
[2022-08-06 22:11:43,415] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:11:43,416] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:11:43,448] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211205T000000, start_date=20220806T221143, end_date=20220806T221143
[2022-08-06 22:11:43,481] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:11:43,529] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:23:01,343] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-05T00:00:00+00:00 [queued]>
[2022-08-06 22:23:01,359] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-05T00:00:00+00:00 [queued]>
[2022-08-06 22:23:01,359] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:01,359] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:23:01,359] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:23:01,377] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2021-12-05 00:00:00+00:00
[2022-08-06 22:23:01,384] {standard_task_runner.py:52} INFO - Started process 1352 to run task
[2022-08-06 22:23:01,386] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-12-05T00:00:00+00:00', '--job-id', '390', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp0oquevyd', '--error-file', '/tmp/tmpo8wk9jnt']
[2022-08-06 22:23:01,386] {standard_task_runner.py:80} INFO - Job 390: Subtask second_task
[2022-08-06 22:23:01,461] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-12-05T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:23:01,544] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-12-05T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-05T00:00:00+00:00
[2022-08-06 22:23:01,545] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:23:01,545] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:23:01,559] {subprocess.py:85} INFO - Output:
[2022-08-06 22:23:01,560] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:23:01,561] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:23:01,594] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211205T000000, start_date=20220806T222301, end_date=20220806T222301
[2022-08-06 22:23:01,637] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:23:01,670] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
